kratos/integration/hexahedron_gauss_lobatto_integration_points.h

```diff
@@ -47,12 +47,19 @@ class KRATOS_API(KRATOS_CORE) HexahedronGaussLobattoIntegrationPoints1
         return 4;
     }
 
+    static const IntegrationPointsArrayType GenerateIntegrationPoints()
+    {
+        const IntegrationPointsArrayType integration_points{{
```

Comment:
@loumalouomega, Please note that these local variable should be static in order to have the initialization once:

```cpp
static const IntegrationPointsArrayType s_integration_points{{
```

Same for all other methods.

Reply:
I think I tried to do that when trying this and I had compilation errors, don't remember what

Reply:
So please try it again to see what was the problem

Reply:
Done, I could not test it until now

Reply:
But is not static yet. (Or I am not seeing the last changes)

My problem is the reallocation of the variable, which can be time consuming (I am not sure if we can count on optimizer magic in this case as it is constant)

Reply:
![image](https://user-images.githubusercontent.com/19991680/94420002-d15feb80-0183-11eb-9349-91b6cd0a334d.png)
outdated

Reply:
![image](https://user-images.githubusercontent.com/8911479/94420649-bb065f80-0184-11eb-8cc9-0a77c30e64f7.png)

Reply:
I was checking the changes

Reply:
OK, but the static was added